.github/workflows/deploy.yml

```diff
@@ -10,6 +10,8 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    env:
+      SLS_DEPRECATION_DISABLE: "*" # Turn off deprecation warnings in the pipeline
```

Comment:
This shuts off the verbose deprecation warnings in the pipelines.  This is a cosmetic change, and just works to make deploy logs easier to read.